dbHelpers/db.js

```diff
@@ -0,0 +1,39 @@
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/sagittarius', {useNewUrlParser: true});
+
+var db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+  console.log('connected to Sagittarius db (similar service)');
+});
+
+var breedSchema = new mongoose.Schema({
+  id: Number,

```

Comment:
You could give id a unique property in your schema so you never get two docs with the same id in the database when querying data by id by maybe accidentally seeding twice.